examples/mock-parsed.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "publicKey": 234234,
```

Comment:
Are we doing anything with the `publicKey` field?

I think using Ethereum address is good enough for our MVP since we are using MetaMask and we can check if the address matches.